tests/test_service.py

```diff
@@ -169,7 +169,7 @@ def test_rsi_metadata(self):
                 self.assertValid(survey)
 
     def test_missing_version(self):
-        survey = json.loads(self.message['0.0.1'])
+        json.loads(self.message['0.0.1'])
```

Comment:
I don't get what this test is testing?

Reply:
The test looks a bit misleading, If I'm right, the `ValidSurveyId` function checks the version number, by what's passed through `def ValidSurveyId(value, version=None):` and this json.loads is not actually getting used.
Might be worth having a look at something like...
```
def test_invalid_version(self):
    survey = json.loads(self.message['5'])
    with self.assertRaises(AttributeError):
        server.ValidSurveyId(survey['survey_id'], survey['version'])

def test_valid_version(self):
    survey = json.loads(self.message['0.0.1'])
    server.ValidSurveyId(survey['survey_id'], survey['version'])
    self.assertTrue?
```